### PR TITLE
Add .NET characteristics

### PR DESCRIPTION
**Pull request by richlander**

### CODE REVIEW

File: docs/about/index.md [hunk 1]
```diff
@@ -27,3 +27,27 @@ Multiple [.NET implementations and products](products.md) are available, based o
 - Learn about [C#](../csharp/)
 - Learn about [F#](../fsharp/)
 - Browse the [.NET API Library](../api)
+- [Introduction to the Common Language Runtime](https://github.com/dotnet/coreclr/blob/master/Documentation/botr/intro-to-clr.md)
+
+Characteristics
+---------------
+
+**Multi-Language** -- .NET provides a well-defined type system, formats, runtime and framework and tools that can be used by multiple languages, both for their own execution and also to interoperate with other languages using those same components of .NET as their shared curency.
```
Comment (mairaw) [inline docs/about/index.md]: typo: curency

File: docs/about/index.md [hunk 2]
```diff
@@ -27,3 +27,27 @@ Multiple [.NET implementations and products](products.md) are available, based o
...
+
+**Managed Memory** -- .NET manages automatically manages memory for you via a garbage collector. It ensures that you always reference live objects, guaranteeing that you avoid nasty problems like buffer overruns and access violations. This includes array bounds checking.
+
+**Type Safety** -- The primary .NET model for functionality and memory representation is "types". Types define shape and optionally behavior. The runtime ensures that calling code can only operate on types according to their defintion and specified visibility of members, providing consistent, reliable and secure results.
```
Comment (mairaw) [inline docs/about/index.md]: typo: defintion

File: docs/about/index.md [hunk 3]
```diff
@@ -27,3 +27,27 @@ Multiple [.NET implementations and products](products.md) are available, based o
...
+
+**User-defined Value Types** -- Value types are a useful category of types since they offer the semantic of "pass by value" instead of "pass by reference", as is the case for classes. Value types are most obviously usefuly for numeric data. .NET enables value types for both primitives types, like integers, and user-defined types.
```
Comment (mairaw) [inline docs/about/index.md]: typo: usefuly
primitives types -> primitive types

File: docs/about/index.md [hunk 4]
```diff
@@ -27,3 +27,27 @@ Multiple [.NET implementations and products](products.md) are available, based o
...
+
+**Generic types** -- Generic types are types with one or more type parameters that can be specified on a per-instantiation basis. This is useful for many types, which otherwise would  expose contents as the Object type or require multiple type definitions. For example, a given instantiation of a collection type can be made specific to People, GPS locations or strings.
+
+**Reflection** -- .NET defines a metadata format that describes the types within a binary. The reflection subsystem uses this data, exposing APIs for both reading and instaniating types at runtime. This facility is very useful for dynamic scenarios where it is not convenient to know the exact implementation of a program ahead of time.
```
Comment (mairaw) [inline docs/about/index.md]: typo: instaniating

File: docs/about/index.md [hunk 5]
```diff
@@ -27,3 +27,27 @@ Multiple [.NET implementations and products](products.md) are available, based o
...
+
+**Flexible code generation** -- .NET does not prescribe a specific approach to transforming .NET binaries into machine code. Many approaches have been used successfully, including interpretation, just-in-time (JIT) compilation, ahead-of-time (AOT) compilation with JIT fall-back and AOT with no JIT fall-back. Each of these strategies can be valuable and there are opportunities for using them together.
```
Comment (mairaw) [inline docs/about/index.md]: fall-back -> fallback

File: docs/about/index.md [hunk 6]
```diff
@@ -27,3 +27,27 @@ Multiple [.NET implementations and products](products.md) are available, based o
...
+
+**Cross-platform** -- .NET was intended to be cross platform from its inception. The binary format and the instruction set are operating system, CPU and pointer-size agnostic. A given .NET binary built in 2000 to run on a 32-bit Windows machine can run on the ARM64 iOS device on 2016 without issue.
+
+**Open source** -- The .NET Core and Mono implemenations of .NET are open source, using the MIT license. These general purpose runtimes can be used as the basis of academic research or teaching or commercial products. The open nature also means that anyone can contribute back to the upstream product code.
```
Comment (mairaw) [inline docs/about/index.md]: typo: implemenations 
The open nature -> The open nature of the source (or any any other noun)?

**Comment by mairaw** [comment]
Other typos:
line 22: symetry and evironment

**Comment by mairaw** [comment]
Change e.g. to for example

**Comment by mairaw** [comment]
The learn about links need to link to a specific file link index.md. Otherwise, our list of warnings for our builds will keep growing.

File: docs/about/index.md (hunk 1 above)
Comment (mairaw) [inline docs/about/index.md]: nit: runtime and framework and tools -> runtime, framework and tools

File: docs/about/index.md [hunk 7]
```diff
@@ -27,3 +27,27 @@ Multiple [.NET implementations and products](products.md) are available, based o
...
+
+**Reflection** -- .NET defines a metadata format that describes the types within a binary. The reflection subsystem uses this data, exposing APIs for both reading and instaniating types at runtime. This facility is very useful for dynamic scenarios where it is not convenient to know the exact implementation of a program ahead of time.
+
+**Flexible code generation** -- .NET does not prescribe a specific approach to transforming .NET binaries into machine code. Many approaches have been used successfully, including interpretation, just-in-time (JIT) compilation, ahead-of-time (AOT) compilation with JIT fall-back and AOT with no JIT fall-back. Each of these strategies can be valuable and there are opportunities for using them together.
+
+**Cross-platform** -- .NET was intended to be cross platform from its inception. The binary format and the instruction set are operating system, CPU and pointer-size agnostic. A given .NET binary built in 2000 to run on a 32-bit Windows machine can run on the ARM64 iOS device on 2016 without issue.
```
Comment (mairaw) [inline docs/about/index.md]: without issue -> without any issues?

File: docs/about/index.md [hunk 8]
```diff
@@ -27,3 +27,27 @@ Multiple [.NET implementations and products](products.md) are available, based o
...
+
+**Open source** -- The .NET Core and Mono implemenations of .NET are open source, using the MIT license. These general purpose runtimes can be used as the basis of academic research or teaching or commercial products. The open nature also means that anyone can contribute back to the upstream product code.
+
+**Standardized** -- .NET is specified via open [ECMA standards](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) that outlines its capabilities and that can be used to make a new implementation. There are other implementations, with Mono and Unity being the most popular beyond the Microsoft ones.
```
Comment (mairaw) [inline docs/about/index.md]: ECMA standards that outlines -> ECMA standards that outline
There are other implementations -> There are other .NET implementations

**Comment by mairaw** [comment]
LGTM modulo the comments I've left

File: docs/about/index.md [hunk 9]
```diff
@@ -27,3 +27,27 @@ Multiple [.NET implementations and products](products.md) are available, based o
...
 - Learn about [F#](../fsharp/)
 - Browse the [.NET API Library](../api)
+- [Introduction to the Common Language Runtime](https://github.com/dotnet/coreclr/blob/master/Documentation/botr/intro-to-clr.md)
+
+Characteristics
+---------------
+
+**Multi-Language** -- .NET provides a well-defined type system, formats, runtime and framework and tools that can be used by multiple languages, both for their own execution and also to interoperate with other languages using those same components of .NET as their shared curency.
+
+**Managed Memory** -- .NET manages automatically manages memory for you via a garbage collector. It ensures that you always reference live objects, guaranteeing that you avoid nasty problems like buffer overruns and access violations. This includes array bounds checking.
```
Comment (svick) [inline docs/about/index.md]: Typo: manages automatically manages → automatically manages

File: docs/about/index.md [hunk 10]
```diff
@@ -19,35 +19,55 @@ About .NET
...
+Multiple [.NET implementations and products](products.md) are available, based on open [.NET Standards](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) that specify the fundamentals of the platform. They are separately optimized for different application types (for example, desktop, mobile, gaming, cloud) and support many chips (e.g. x86/x64, ARM) and operating systems (e.g. Windows, Linux, iOS, Android, macOS). Open source is also an important part of the .NET ecosystem, with multiple .NET implementations and many libraries available under OSI-approved licenses.
 
 - Learn about [.NET](../standard/index.md)
-- Learn about [C#](../languages/csharp/index.md)
-- Learn about [F#](../languages/fsharp/index.md)
-- Browse the [.NET API Library](../api)
+- Learn about [C#](../csharp/index.md)
+- Learn about [F#](../fsharp/index.md)
```
Comment (svick) [inline docs/about/index.md]: Why did you change the C# and F# links? The path with `languages` should be correct.

Comment (mairaw) [inline docs/about/index.md]: If you run docfx (or in this case open publishing), you get lots of warnings about broken links. I'd like to get those fixed throughout the content so I suggested he changed to that. I'll work with the team to get that improved and then we can change the links.

Comment (richlander) [inline docs/about/index.md]: Because we're going to unwrap "languages" later today.
